internal/api/handlers/profiles_test.go

```
@@ -0,0 +1,88 @@
+		{
+			name: "unsupported encoding",
+			req: models.ProfileRequest{
+				ContentB64: encoded,
+				Encoding:   "gzip",
+			},
+			wantErr: errProfileEncodingUnsupported,
+		},
+	}
+
+	for _, tt := range tests {
+		tt := tt
```

golangci-lint (2.12.1)

[error] The copy of the 'for' variable "tt" can be deleted (Go 1.22+)

(copyloopvar)